test.js

```diff
@@ -252,19 +266,19 @@ test('fileType.mimeTypes.has', t => {
 
 test('validate the input argument type', t => {
 	t.throws(() => {
-		fileType('x');
+		fileType.fromBuffer('x');
 	}, /Expected the `input` argument to be of type `Uint8Array`/);
 
 	t.notThrows(() => {
-		fileType(Buffer.from('x'));
+		fileType.fromBuffer(Buffer.from('x'));
 	});
 
 	t.notThrows(() => {
-		fileType(new Uint8Array());
+		fileType.fromBuffer(new Uint8Array());
 	});
 
 	t.notThrows(() => {
-		fileType(new ArrayBuffer());
+		fileType.fromBuffer(new ArrayBuffer());
```

Comment:
```suggestion
		FileType.fromBuffer(new ArrayBuffer());
```

For consistency with the readme.

Reply:
Done